index.js

```diff
@@ -77,21 +94,22 @@ genderCount(data);
 // ***************************************************************************
 // Bonus - Iteration 6
 // ***************************************************************************
-
 const promo20 = users => {
-  // Your code goes here ...
+  users.filter(item => item.balance.replace('$', "").replace(',', '')
+  > 20000 && console.log (`Dear ${item.firstName} ${item.lastName} since your balance is ${item.balance}, you are eligible to apply for this awesome credit card.`))
 };
 
+
 // expected output:
 // Dear Howard, since your balance is $21,307.75, you are eligible to apply for this awesome credit card.
 // Dear Rachelle, since your balance is $35,121.49, you are eligible to apply for this awesome credit card.
 
 // ***************************************************************************
 // Bonus - Iteration 7
 // ***************************************************************************
-
 const addActive = users => {
   // Your code goes here ...
+  users.forEach(user => user.isActive = true)
 };
```

Comment:
This function immediately is changing the values you are passing onto it. 

In functions, unless said otherwise, try to create a best practice that you return a totally new value.
For example
```js
const addActive = users => {
return users.map(user => {
return {
...user, //here you are getting the whole user object copied
isActive: true
}
})
}
```

After this function you get an array of every user active but you get a totally new value.